.github/workflows/validate_apps.yml

```
@@ -0,0 +1,25 @@
+name:  Validate Links
+on:
+  workflow_dispatch:
+
+jobs:
+  validate-links:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: 'npm'
+
+      - name: Install Dependencies
+        run: npm ci
+  
+      - name: Run validation
+        run: npm run validate:links
+        env:
+           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
        run: npm run validate:link
```

validate link does not need a gh token, PR mode needed

---

ok thanks tony!